internal/pkg/skuba/kubeadm/configmap.go

```diff
@@ -60,7 +60,7 @@ func GetCurrentClusterVersion() (*version.Version, error) {
 	if err != nil {
 		return nil, err
 	}
-	return version.MustParseSemantic(initCfg.KubernetesVersion), nil
+	return version.ParseSemantic(initCfg.KubernetesVersion)
```

Comment:
nil missing

Reply:
`ParseSemantic` returns (version, err) so this is just passing through